### ISSUE TITLE
PracticeLocations

### ISSUE THREAD

**Issue by ghost1face** [ForcuraCo/npi-api-net#19]
New field: `practiceLocations` was added to the response.  Interpret and handle this field accordingly.

It looks like all fields are now defined in an easy to read table here:  https://npiregistry.cms.hhs.gov/help-api/json-conversion

**Comment by tiesont** [comment]
@ghost1face Is this and the other three issues still something you're looking for help with? I was actually just about to implement a .NET6 client for the API and then found yours, so I'd be happy to help (most of these look minor and/or already in progress?)

**Comment by ghost1face** [comment]
#19 and #16 would be great to start. Any contributions are welcome!

**Comment by tiesont** [comment]
> #19 and #16 would be great to start. Any contributions are welcome!

Awesome! I see that there were already branches for those tasks, here. I pulled them into my fork and merged - they seem to work as-is. I imagine it's just a bit of cleanup at this point (like removing the conditional package references for unsupported frameworks)?

**Comment by ghost1face** [comment]
Wow, looks like I started it and never came back. Yes, if you'd like to start with those and help validate that would be great. Maybe there's something I missed in there as well

**Comment by tiesont** [comment]
This is actually pretty easy to implement, but needs either a NPI that is known to have practice locations in the result, or the test(s) for this feature would have to accept something like https://github.com/richardszalay/mockhttp to allow supplying a "fake" response with that property populated.

Want me to work on this?

**Comment by ghost1face** [comment]
Yeah, feel free.  I'd love to test with a real scenario if possible.  This way we can validate what the response looks like.  The NPPES API has burned us all several times where they make a breaking change with 0 warning or the actual response doesn't match documentation.

So anything to make sure we align with what they actually return would be best IMO
